package.json

```diff
@@ -34,6 +34,7 @@
     "@leafygreen-ui/checkbox": "^4.1.1",
     "@leafygreen-ui/code": "^3.5.0",
     "@leafygreen-ui/icon": "^6.4.2",
+    "@reach/router": "^1.3.4",
```

Comment:
Curious about your decision to add `"@reach/router"` here instead of using your current solution for routing. Was this something you couldn't do with your current implementation?

Reply:
So [`reach/router` is already a co-dependency of Gatsby](https://www.gatsbyjs.com/docs/reach-router-and-gatsby/) meaning we already had it installed as a dependency but are now just indicating it is not a dev one (lockfile change is super small too). Gatsby gives a `location` prop to each [page component,](https://github.com/mongodb/devhub/pull/520) so the learn page component has a `location` prop provided but we also want the `CardList` component to have this as well so it can add the query param.

I wanted to go with this approach of adding the hook instead of passing the location down since it made the props a bit cleaner and then we wouldn't have to go back and update every `CardList` with a location.

So `reach/router` just makes things a bit cleaner instead of prop drilling but in theory we don't need it.

Reply:
makes sense! thank you for the explanation